### DISCUSSION TITLE
Modify a survey's colors and fonts

### DISCUSSION THREAD

**Discussion by steffejr** [jspsych/jsPsych#2989] [General]
Hello, I have made a survey with multi-select and three text boxes. I need them to appear all on the same page. I used jsPsychSurvey to do this and it works fine. However, I would like to customize the background color and the fonts to conform with the rest of my tasks. May I have some guidance on how to do this, please?
I have included this CSS in my html:
    <link rel="stylesheet" href="https://unpkg.com/@jspsych/plugin-survey@0.2.1/css/survey.css">
I find this CSS so complicated that I cannot make any headway into modifying my survey.
Any guidance is greatly appreciated.

**Comment by ChristopheBossens** [comment]
For issues such as these I usually launch into the developer mode (F12 on most browsers), and inspect the elements to see what css classes have been assigned to them. And then I override these properties in the style section of my HTML file. 

To change the background color, I used the descendant selector for the `.sv_main` class, so that it is applied to all div elements that follow (otherwise I could not get it to work :D).

This is my head section (you might have different syntax for importing some of the jsPsych stuff):

```
<head>
    <title>My experiment</title>
    <script src="../jspsych73/jspsych.js"></script>
    <link href="../jspsych73/jspsych.css" rel="stylesheet" type="text/css" />
    <script src="../jspsych73/plugin-survey.js"></script>
    <link rel="stylesheet" href="https://unpkg.com/@jspsych/plugin-survey@0.2.1/css/survey.css">
    <style>
        .sv_main div{
            background-color: aquamarine;
        }

        .sv_main {
            font-family:'Courier New', Courier, monospace;
            font-size: 16px;
            text-align: center;
        }
    </style>
  </head>
```
I tested it with the first example on[ this page](https://www.jspsych.org/7.3/plugins/survey/#examples). Is this what you need?

**Comment by steffejr** [comment]
That is perfect. Thank you very much.
I may be pushing it, but ... I have a question based on the attached screen
shot. Is there any chance you know how I can get "Intrusion?" and the text
box to be in the same line as each other? And then, some way to remove the
large space between the two intrusion lines? I want to make things as
compact as possible.
Thank you.

 [image: Screen Shot 2023-03-23 at 9.43.40 AM.png]

**Comment by ChristopheBossens** [comment]
I probably can but I cannot see the image so I do not really know what you mean :). Can you share the image in another way?
